x-pack/plugins/security_solution/public/detections/components/rules/select_rule_type/index.tsx

```diff
@@ -29,7 +29,7 @@ interface SelectRuleTypeProps {
   isUpdateView: boolean;
 }
 
-export const SelectRuleType: React.FC<SelectRuleTypeProps> = ({
+export const SelectRuleTypeComponent: React.FC<SelectRuleTypeProps> = ({
```

Comment:
```suggestion
export const SelectRuleTypeComponent: React.FC<SelectRuleTypeProps> = memo({
```

nit: I've been doing the `memo` inline as of late so that way the DX for `find usages` within IDE's is a bit better (taking you to the top of the file, and also being able to do find usages from the main component export as well instead of having to scroll to the bottom and do usages from there). I heard there may be some fallout w/ component naming within the React Component view of browser dev tools, but I haven't experienced that personally.